### PR TITLE
Deprecate and remove Xunit1 Testkit project

### PR DESCRIPTION
**Pull request by alexvaluyskiy**
XUnit2 was released two years ago, I think it is a time to remove the package for Xunit1

### CODE REVIEW

**Comment by Aaronontheweb** [comment]
Don't we need to rename the XUnit2 nuget package then?

**Comment by Aaronontheweb** [comment]
errr, let me rephrase both the XUnit 1 and XUnit2 packages should now have the same content as the XUnit2 one, correct?

**Comment by alexvaluyskiy** [comment]
@Aaronontheweb the last version of XUnit 1.x was released almost 4 years ago. Why we should support it in 2017?

**Comment by Aaronontheweb** [comment]
@alexvaluyskiy what I meant was: just have both packages start serving up XUnit2 instead. So effectively make the Akka.TestKit.XUnit project serve up the same material as XUnit2. Make sense?
